src/config/permissions.json

```diff
@@ -0,0 +1,26 @@
+{
+  "admin": {
+    "accessReviews": true,
+    "accessUsers": true,
+    "resetReviews": true,
+    "accessTransactions": true,
+    "accessAdminNav": true,
+    "editAdminTranscripts": true
```

Comment:
I don't understand the chosen name for this permission.

What does it mean to "edit admin transcripts"? Good names help with readability.

In order to come up with a good name, you should start with the question: "what this permission allows?"
This permission allows the user to submit to their fork of the repository instead of the upstream repo.
So a good name could be "**submitToOwnRepo**"

Reply:
Okay. Yh